src/util.cpp

```diff
@@ -28,22 +28,22 @@
 
 /* Implementation *************************************************************/
 // Input level meter implementation --------------------------------------------
-void CStereoSignalLevelMeter::Update ( const CVector<short>& vecsAudio,
+void CStereoSignalLevelMeter::Update ( const CVector<float>& vecsAudio,
                                        const int             iMonoBlockSizeSam,
                                        const bool            bIsStereoIn )
 {
     // Get maximum of current block
     //
     // Speed optimization:
     // - we only make use of the negative values and ignore the positive ones (since
-    //   int16 has range {-32768, 32767}) -> we do not need to call the fabs() function
+    //   float has range {-1.0f, 1.0f}) -> we do not need to call the fabs() function
```

Comment:
Well, float has not the range -1 to 1. We define it to be the range would be more accurate.

Reply:
Fixed.